package.json

```diff
@@ -88,22 +112,51 @@
             "verbose"
           ],
           "default": "off",
-          "description": "Traces the communication between VS Code and the XML language server.",
+          "markdownDescription": "Traces the communication between VS Code and the XML language server in the Output view. Default is `off`.",
           "scope": "window"
         },
+        "xml.logs.client": {
+          "type": "boolean",
+          "default": true,
+          "markdownDescription": "Enable/disable logging to the Output view. Default is `true`."
+        },
         "xml.catalogs": {
           "type": "array",
           "default": [],
           "items": {
             "type": "string"
           },
-          "description": "Array of XML Catalogs.",
+          "markdownDescription": "Register XML catalog files. See how to configure [XML catalog with XSD](https://github.com/redhat-developer/vscode-xml/wiki/XMLValidation#xml-catalog-with-xsd) or [XML catalog with DTD](https://github.com/redhat-developer/vscode-xml/wiki/XMLValidation#xml-catalog-with-dtd) for more information.",
           "scope": "window"
         },
-        "xml.logs.client": {
+        "xml.fileAssociations": {
+          "type": "array",
+          "default": [],
+          "items": {
+            "type": "object",
+            "properties": {
+              "systemId": {
+                "type": "string",
+                "description": "The path or URL to the XML schema (XSD or DTD)."
+              },
+              "pattern": {
+                "type": "string",
+                "markdownDescription": "File glob pattern. Example: `**/*.Format.ps1xml`\n\nMore information on the glob syntax: https://docs.oracle.com/javase/tutorial/essential/io/fileOps.html#glob"
+              }
+            },
+            "required": [
+              "systemId",
+              "pattern"
+            ]
+          },
+          "markdownDescription": "Allows XML schemas/ DTD to be associated to file name patterns.\n\nExample:\n```json\n[{\n  \"systemId\": \"path/to/file.xsd\",\n  \"pattern\": \"file1.xml\"\n},\n{\n  \"systemId\": \"http://www.w3.org/2001/XMLSchema.xsd\",\n  \"pattern\": \"**/*.xsd\"\n}]\n```",
+          "scope": "window"
+        },
+        "xml.format.enabled": {
           "type": "boolean",
           "default": true,
-          "description": "Should the server log to client output."
+          "markdownDescription": "Enable/disable ability to format document. Default is `true`. See [here](https://github.com/redhat-developer/vscode-xml/wiki/Formatting#xmlformatenabled-) for more information.",
```

Comment:
I don't think we need a link here, since the wiki says the same thing as the description.

Reply:
fixed